spec/support/fixtures/test_models.rb

```diff
@@ -43,6 +43,7 @@ def public_identifier
 
   def self.setup_models
     conn = ClientRequest.connection
+    return if conn.table_exists? "test_client_requests"
     conn.create_table(:test_client_requests, force: true) do |t|
```

Comment:
Actually, I'm guessing that my new line is redundant. I hadn't noticed `force: true`.